harvest/algo.py

```diff
@@ -220,7 +220,7 @@ def default_param(self, symbol, interval, ref, prices):
             if interval == None:
                 interval = self.trader.interval
             if prices == None:
-                prices = self.trader.store.load(symbol, interval)
+                prices = self.trader.storage.load(symbol, interval)
```

Comment:
do you want this to be called `storage` or `store`. I prefer `storage` because `store.store` becomes weird and `store` is a function of `storage.`

Reply:
Actually I agree, `store` sounds too much like a verb, it should be `storage`